<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::WebInspector</class>
 <widget class="QWidget" name="GammaRay::WebInspector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>530</height>
   </rect>
  </property>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <widget class="QComboBox" name="webPageComboBox"/>
        </item>
        <item>
         <widget class="QWebInspector" name="webInspector" native="true"/>
        </item>
       </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
